spec/koans.js

```diff
@@ -126,30 +132,28 @@ describe('a template string, is wrapped in ` (backticks) instead of \' or ". ',
 describe('arrow functions. ', () => {
 
   it('are shorter to write', function() {
-    let func = () => {
-      /*........*/
-    };
-    //expect(func()).toBe('I am func');
+    let func = () => 'I am func';
+    expect(func()).toBe('I am func');
   });
 
   it('a single expression, without curly braces returns too', function() {
-    /*let func = () => .........;*/
-    //expect(func()).toBe('I return too');
+    let func = () => 'I return too';
+    expect(func()).toBe('I return too');
   });
 
   it('one parameter can be written without parens', () => {
-   /* let func = ........;*/
-    //expect(func(25)).toEqual(24)
+    let func = () => 24;
+    expect(func(25)).toEqual(24)
   });
 
   it('many params require parens', () => {
-    /* let func = ........;*/
-    //expect(func(23,42)).toEqual(23+42)
+    let func = () => (23+42);
```

Comment:
Here the objective would be to do something like ``` (a, b) => a + b ``` to demonstrate that various parameters require parens